plugins/plugin-git/che-plugin-git-ext-git/src/main/java/org/eclipse/che/ide/ext/git/client/fetch/FetchPresenter.java

```diff
@@ -172,12 +182,50 @@ public void onFetchClicked() {
             })
         .catchError(
             error -> {
+              if (getErrorCode(error.getCause()) == UNAUTHORIZED_GIT_OPERATION) {
```

Comment:
I see such code at least few times. Can we have a method anywhere to perform such logic?